web/src/utils/analytics.ts

```diff
@@ -16,5 +16,11 @@ export default function trackEvent(
   eventId: string,
   additionalProps: PlausibleEventProps = {}
 ): void {
+  const isDevelopment = import.meta.env.MODE === 'development';
+  if (isDevelopment) {
+    console.log("not sending event to plausible because we're not in production");
+
+    return;
+  }
```

Comment:
If the change away from Plausible `.local.script` is fine, then we don't really need this, right?